lib/ckb/wallet.rb

```diff
@@ -131,7 +129,8 @@ def deposit_to_dao(capacity, key: nil)
       i = gather_inputs(
         capacity,
         output.calculate_min_capacity,
-        change_output.calculate_min_capacity
+        change_output.calculate_min_capacity,
+        0
```

Comment:
Is it possible to set a default value for the fee?

Reply:
`gather_inputs` is an inner method, I think it doesn't matter.